migration/wcmp-upgrade-migration-v4-2-1.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+use migration\WCMP_Upgrade_Migration;
+
+if (! defined('ABSPATH')) {
+    exit;
```

Comment:
```suggestion
    exit();
```

Reply:
Niet mee eens, in alle bestanden die we hebben wordt `exit` gebruikt. Ik zie dat woocommerce het zo ook definieert.